internal/packing/packing.go

```diff
@@ -178,7 +152,15 @@ func alloc(n *Node, width, height int) *Node {
 				parent: n,
 			}
 		}
-		return alloc(n.child0, width, height)
+		// Note: it now MUST fit, as child0.width >= width and child0.height >= height.
+		// Otherwise, this function would have already failed right at the top.
+		// Thus, alloc can't return nil, but it may do another split along the other dimension
+		// to get an exactly width x height node.
```

Comment:
> it now MUST fit

Does `fit` mean that `child0` has enough capability to allocate a new node (width, height)?

> child0.width >= width and child0.height >= height

I suggested we could write this as an assertion in Go.

> width x height node

Does this mean "a node with a size (width, height)"?

Reply:
Applied.